### ISSUE TITLE
add readme explaining how to get started with this app

### ISSUE THREAD

**Issue by camallen** [zooniverse/text-editor#14]
Flesh out the readme to explain how to run tests and smoke test this application from scratch. 

ideally it'll use docker to build the app / deps and then docker-compose to run the app locally with all it's dependencies built it.

**Comment by eatyourgreens** [comment]
https://github.com/zooniverse/zoo-ml-subject-assistant has a simple Docker setup that you can probably copy. The web app is the `dev` service in the docker-compose file. You can ignore the `proxy` service, which sets up a simple API proxy server.

**Comment by wgranger** [comment]
Good call. I'll add a couple quick yarn commands in the meantime until I can get Docker setup.
